### ISSUE TITLE
Introduction to Neural Networks and Computer Vision || Workshop - 3

### ISSUE THREAD

**Issue by DolceParadise** [Udyam/YouTubeVideoTimestamps#2]
https://youtu.be/GAEcxIuIPuA (Video Link)

0:00 Last Workshop
1:20 Problem with Using Linear Hypothesis
2:50 Polynomial Regression
5:57 What is a Neural Network?
10:57 Notations
12:43 Calculating Features of Hidden Layer
20:20 Backpropagation
33:00 What is Deep Learning?
33:46 Sigmoid Activation 
34:06 Softmax Activation  
35:10 Linear Activation 
35:22 ReLU
36:08 Optimizers
37:30 Implementation

**Comment by pr-mittal** [comment]
Thank You . The timestamps have been updated
